### ISSUE TITLE
Limit a specific tier to Indivudals or Organizations

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#7358]
From: https://opencollective.slack.com/archives/C1974QBNK/p1712831846757319

> Is there any way to say that a tier can only be available to individual rather than organisational accounts? Use case is we have individual and corporate tiers, and every now and then companies try to sneak in at the individual tiers :slightly_smiling_face:

This seems like a challenge for unauthenticated users, since we can't know in advance whether they're individuals or organizations - we'd have to ask in the contribution flow.

**Comment by RCheesley** [comment]
Yes, it's also a huge problem for us when we get a guest joining because we have absolutely no way of being able to contact them as an org, especially - I end up having to google search the name, go via linkedin/contact pages to try to figure out the point of contact. So ideally also being able to say you have to have an account would be helpful, too.

**Comment by Betree** [comment]
@RCheesley Sound like your issue with guests is more of an issue with "incognito", since guest emails are normally available in the contributors CSV?

**Comment by RCheesley** [comment]
Yes, indeed!
